src/core/lib/slice/slice_internal.h

```diff
@@ -168,9 +168,12 @@ struct grpc_slice_refcount {
   void* destroy_fn_arg_ = nullptr;
 };
 
+static_assert(std::is_trivially_destructible<grpc_slice_refcount>::value,
```

Comment:
I don't think we need this assertion, since we have the one for `kNoopRefcount` elsewhere.  This check is really about the instance being defined as a global variable, not about the class itself.

Reply:
Removed.